### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jharenza**
Add instructions for repo cloning and data download

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,9 +27,14 @@ docker pull pgc-images.sbgenomics.com/zhuangzhuanggeng/d3b_hope_analysis:latest
 
 To run the docker, run the command line below. For mac M1 user, add `--platform=linux/arm64`.
 ```
-docker run -d -e PASSWORD=pass -p 8787:8787 --name <CONTAINER_NAME> -v $PWD:/home/rstudio/HOPE pgc-images.sbgenomics.com/zhuangzhuanggeng/d3b_hope_analysis:latest
+docker run -d -e PASSWORD=pass -p 8787:8787 --name <CONTAINER_NAME> -v $PWD:/home/rstudio/hope-cohort-analysis pgc-images.sbgenomics.com/zhuangzhuanggeng/d3b_hope_analysis:latest
+```
```
Comment (JosephDybas) [inline README.md:32]: Should the docker exec command be added?

docker exec -ti hope-cohort-analysis bash

File: README.md [hunk 2]
```diff
@@ -14,9 +27,14 @@ docker pull pgc-images.sbgenomics.com/zhuangzhuanggeng/d3b_hope_analysis:latest
...
 
+## Navigate to the repository root
 ```
+cd /home/rstudio/hope-cohort-analysis
+```
+
 
 ## Modules
```
Comment (JosephDybas) [inline README.md:40]: "proteomic-data" is not included in the modules list. Should it be added along with a short description below the list?

Comment (JosephDybas) [inline README.md:40]: "7) `tmb-calculation`: Adapted from [OpenPedCan-anaysis](https://github.com/d3b-center/OpenPedCan-analysis/tree/dev/analyses/tmb-calculation)"
This line refers to something I think no longer exists and is not in the list of modules.

Comment (JosephDybas) [inline README.md:40]: "molecular-subtyping-HGG" module is missing the short descriptions below the list.

Comment (JosephDybas) [inline README.md:40]: "tp53_nf1_score" module is missing the short descriptions below the list.

File: README.md [hunk 3]
```diff
@@ -4,6 +4,19 @@ Module authors: [Komal S. Rathi](https://github.com/komalsrathi/),
...
+To clone the repository, run:
+```
+git clone git@github.com:d3b-center/hope-cohort-analysis.git
+```
+
+## Download the data
+
+To download the current data release:
+```
+bash download_data.sh
```
Comment (JosephDybas) [inline README.md:18]: The file name is "download-data.sh" with a hyphen rather than underscore.

**Review by JosephDybas — CHANGES_REQUESTED** [review]
I ran the commands as written in the README.md file. The repo cloned properly and the Docker container was established. I added a few comments which can be addressed. Thanks.

File: README.md (hunk 2 above)
Comment (jharenza) [reply README.md:40]: thanks - can you add an issue for these to be added, as this wasn't the scope of the PR? Thanks!

File: README.md (hunk 1 above)
Comment (jharenza) [reply README.md:32]: I can add, yes

File: README.md (hunk 3 above)
Comment (jharenza) [reply README.md:18]: good catch!

**Review by JosephDybas — APPROVED** [review]
Changes look good to me. Thanks!

**Pull request merged**